pipeline/extract.py

```diff
@@ -95,14 +95,55 @@ def scrape_pricing_process(html_content: bytes, url: str, product_id: int) -> di
     if "https://store.steampowered.com" in website_url:
         return scrape_from_steam_html(html_content, url, product_id)
 
-    if "https://www.amazon.com" in url or "https://www.amazon.co." in website_url:
-        return scrape_from_amazon_html(html_content, url, product_id)
+    # if "https://www.amazon.com" in url or "https://www.amazon.co." in website_url:
+    #     return scrape_from_amazon_html(html_content, url, product_id)
+
+    if "https://www.debenhams.com" in website_url:
+        return scrape_from_debenhams_html(html_content, url, product_id)
 
     logging.error(
         "Cannot scrape that URL, since it's not an Amazon/Steam webpage.")
     return
 
 
+def scrape_from_debenhams_html(html_content: bytes, url: str, product_id: int) -> dict:
```

Comment:
Thank you so much for adding in this function to scrape from Debenhams, this is so useful!